test/bin/build_images.sh

```diff
@@ -14,29 +14,62 @@ shopt -s nullglob
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
+# shellcheck source=test/bin/get_crel_version_repo.sh
+source "${SCRIPTDIR}/get_crel_version_repo.sh"
+
+test_envsubst() {
```

Comment:
I like this idea. I would split it from the new test to make reviews easier. See also https://issues.redhat.com/browse/USHIFT-1546

Reply:
Sure, I'll create separate PR for this one.

Re templating: Unless we'd like to address "potential non-existence of current release (e.g. early 4.15)" in another way, then we have additional requirement from a templating engine to tell us if the template could not be fully rendered (it would be nice if it was separate from an error about syntax problems etc).

Because this is what I need here: if could not fully render (because CREL_VERSION will be empty (!) in early 4.15) then I don't want to load this osbuild source and don't want the test to even run

Reply:
On the other hand: given our tests will only grow and we introduce optional behavior such as this, should we start thinking about moving away from bash? Logs from bash -x are like last resort because of poor readability. With Python we could replace script by script. I wish we had a log from infrastructure part that looks like robot's log.html